eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Translation.hs

```diff
@@ -116,14 +117,9 @@ instance TranslateEra AlonzoEra VState where
 instance TranslateEra AlonzoEra PState where
   translateEra _ PState {..} = pure PState {..}
 
-instance TranslateEra AlonzoEra CertState where
-  translateEra ctxt ls =
-    pure
-      CertState
-        { certDState = translateEra' ctxt $ certDState ls
-        , certPState = translateEra' ctxt $ certPState ls
-        , certVState = translateEra' ctxt $ certVState ls
-        }
+-- TODO: think about this
+instance CertState AlonzoEra ~ ShelleyCertState AlonzoEra => TranslateEra AlonzoEra ShelleyCertState where
```

Comment:
You should not need this constraint since this import `import Cardano.Ledger.Alonzo.CertState ()` should bring in the type family that satisfy this equality
```suggestion
instance TranslateEra AlonzoEra ShelleyCertState where
```